github-merge.py

```diff
@@ -191,15 +191,20 @@ def make_acks_message(head_commit, acks) -> str:
 def print_merge_details(pull, title, branch, base_branch, head_branch, acks, message):
     print('%s#%s%s %s %sinto %s%s' % (ATTR_RESET+ATTR_PR,pull,ATTR_RESET,title,ATTR_RESET+ATTR_PR,branch,ATTR_RESET))
     subprocess.check_call([GIT,'log','--graph','--topo-order','--pretty=format:'+COMMIT_FORMAT,base_branch+'..'+head_branch])
+    warnings = 0
     if acks is not None:
         if acks:
             print('{}ACKs:{}'.format(ATTR_PR, ATTR_RESET))
-            for name, message in acks.items():
-                print('* {} {}({}){}'.format(message, ATTR_NAME, name, ATTR_RESET))
+            for name, msg in acks.items():
+                if msg is not None and '@' in msg:
+                    warnings += 1
+                print('* {} {}({}){}'.format(msg, ATTR_NAME, name, ATTR_RESET))
         else:
             print('{}Top commit has no ACKs!{}'.format(ATTR_WARN, ATTR_RESET))
     if message is not None and '@' in message:
         print('{}Merge message contains an @!{}'.format(ATTR_WARN, ATTR_RESET))
+    if warnings > 0:
+        print('{}{} merge ACK message(s) contain(s) an @!{}'.format(ATTR_WARN, warnings, ATTR_RESET))
```

Comment:
The message should already contain the ACKs, thus should already warn about that, no?

https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/35eb2c71d09993eb7e87febec0e08aec347b8e7c/github-merge.py#L375-L386

Reply:
The main fix in this PR (the first commit) was added to #53. I don't know if the additional checks are useful now.